client/src/related/productCard.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import StarRating from '../review/starRating.jsx';
+import SalePrice from './salePrice.jsx';
+
+
+class ProductCard extends React.Component {
+  constructor(props) {
+    super(props);
+    //cardType
+    //productInfo
+    //onClick
+    this.state = {
+    };
+  }
+
+  render() {
+    var iconSrc, onIconClick;
+    if (this.props.cardType === 'related') {
+      iconSrc = 'https://cdn3.iconfinder.com/data/icons/sympletts-free-sampler/128/star-512.png';
+    } else {
+      iconSrc = 'https://i.dlpng.com/static/png/6978437_preview.png';
+    }
+    return (
+      <div className="related_productCard" onClick={(event) => {
+        event.preventDefault();
+        this.props.handleCardClick(this.props.productInfo.id);
+      }}>
+        <img className='related_productCardImage' src = {this.props.productInfo.image || 'https://static.thenounproject.com/png/1103191-200.png'}></img>
```

Comment:
I am assuming that the hardcoded address is to handle any missing data?

Reply:
Correct! There was an instance in the API where there was no photos listed for a product, so I chose a "not found" type icon as a backup to make sure it didn't break anything- probably something we should all norm on though before finalizing things!